src/BaGetter.Core/Configuration/BaGetterOptions.cs

```diff
@@ -8,6 +8,12 @@ public class BaGetterOptions
     /// </summary>
     public string ApiKey { get; set; }
 
+    /// <summary>
+    /// The API Key required to authenticate package
+    /// operations. If empty, package operations do not require authentication.
+    /// </summary>
+    public string[] ApiKeys { get; set; }
```

Comment:
These just add additional API keys the admin can give to other teams in order to late be able to revoke specific keys, right? Or did I miss something?